<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HelpDlg</class>
 <widget class="QDialog" name="HelpDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnRemoveFileInDir">
       <property name="text">
        <string>RemoveFileInDir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReName">
       <property name="text">
        <string>ReNameHelpDirWithFile</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="actRenameFile">
       <property name="text">
        <string>RenameFileNameInDir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCopyXml">
       <property name="text">
        <string>CopyXML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear Output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUpdate">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEncry">
       <property name="text">
        <string>Encry Strings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnProductEx">
       <property name="text">
        <string>product_extend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGernerate">
       <property name="text">
        <string>Generate Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnToRcc">
       <property name="text">
        <string>ToRcc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="txtEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
